client/signup/steps/design-picker/preview-toolbar.jsx

```diff
@@ -67,6 +85,14 @@ DesignPickerPreviewToolbar.propTypes = {
 	showDeviceSwitcher: PropTypes.bool,
 	// Called when a device button is clicked
 	setDeviceViewport: PropTypes.func,
+	// Show iframe site address bar
+	showSiteAddressBar: PropTypes.bool,
+	// Filter devices to show in device switcher
+	devicesToShow: PropTypes.array,
+};
+
+DesignPickerPreviewToolbar.defaultProps = {
+	showSiteAddressBar: true,
 };
```

Comment:
Is there any advantage to assigning `.defaultProps` here as opposed to destructuring with a default value above?